arbnode/batch_poster.go

```diff
@@ -640,10 +640,14 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context) (bool, error)
 	if b.batchReverted.Load() {
 		return false, fmt.Errorf("batch was reverted, not posting any more batches")
 	}
-	nonce, batchPosition, err := b.dataPoster.GetNextNonceAndMeta(ctx)
+	nonce, batchPositionBytes, err := b.dataPoster.GetNextNonceAndMeta(ctx)
 	if err != nil {
 		return false, err
 	}
+	var batchPosition batchPosterPosition
+	if err := rlp.DecodeBytes(batchPositionBytes, &batchPosition); err != nil {
+		return false, err
```

Comment:
Could we use fmt.Errorf to wrap the error here and indicate that it was an issue decoding the batch poster position?

Reply:
Done.

Reply:
I think you might have forgotten to push this change up

Reply:
oops, thanks for spotting.